x-pack/plugins/ml/package.json

```diff
@@ -5,8 +5,7 @@
   "private": true,
   "license": "Elastic License 2.0",
   "scripts": {
-    "build:apiDocScripts": "cd server/routes/apidoc_scripts && ../../../../../../node_modules/.bin/tsc",
-    "generateHeader": "node server/routes/apidoc_scripts/target/header_generator.js",
-    "apiDocs": "yarn build:apiDocScripts && yarn generateHeader && cd ./server/routes/ && ../../../../../node_modules/.bin/apidoc-markdown --parse-workers apischema=./apidoc_scripts/target/schema_worker.js --parse-parsers apischema=./apidoc_scripts/target/schema_parser.js --parse-filters apiversion=./apidoc_scripts/target/version_filter.js -i . -o ../routes_doc/ML_API.mdx --header ./apidoc_scripts/header.md --template ./apidoc_scripts/template.md"
+    "generateHeader": "node scripts/apidoc_scripts/header_generator/index.js",
+    "apiDocs": "yarn generateHeader && cd ./scripts/apidoc_scripts/ && ../../../../../node_modules/.bin/apidoc-markdown -i ../../server/routes -c ./apidoc.json -o ./ML_API.mdx --parse-workers apischema=./schema_worker/index.js --parse-parsers apischema=./schema_parser/index.js --parse-filters apiversion=./version_filter/index.js --header ./header.md --template ./template.md"
```

Comment:
The `apidoc.json` file still contains the hardcoded `v8.0.0` version text which is outputted to the top of the mdx file. Is it possible to generate this on-the-fly, or if not, just update to `v.8.8.0`.

Reply:
Added support for the dynamic version based on the current Kibana version in https://github.com/elastic/kibana/pull/153366/commits/59972b89933398fc60e00e873907b27b15840b6d